modules/library/referencing/src/main/java/org/geotools/referencing/operation/matrix/Matrix1.java

```diff
@@ -224,4 +249,137 @@ public Matrix1 clone() {
+                    + values.length + ".  " + "The dimensions of the matrix is 1 by 1.");
+        }
+        m00 = values[0];
+    }
+
+    @Override
+    public void setColumn(int column, double... values) {
+        if (values.length != 0) {
+            throw new IllegalArgumentException("Call setColumn received an array of length "
+                    + values.length + ".  " + "The dimensions of the matrix is 1 by 1.");
+        }
+        m00 = values[0];
+    }
+
+    @Override
+    public void add(double scalar) {
+        m00 += scalar;
+    }
+
+    @Override
+    public void add(double scalar, XMatrix matrix) {
+        if (matrix.getNumRow() != SIZE || matrix.getNumCol() != SIZE) {
+            throw new IllegalArgumentException(Errors.format(ErrorKeys.ILLEGAL_MATRIX_SIZE));
+        }
+        m00 = scalar + matrix.getElement(0, 0);
+    }
+
+    @Override
+    public void add(XMatrix matrix) {
+        if (matrix.getNumRow() != SIZE || matrix.getNumCol() != SIZE) {
+            throw new IllegalArgumentException(Errors.format(ErrorKeys.ILLEGAL_MATRIX_SIZE));
+        }
+        m00 += matrix.getElement(0, 0);
+    }
+    
+    @Override
+    public void add(XMatrix matrix1,XMatrix matrix2) {
+        if (matrix1.getNumRow() != SIZE || matrix1.getNumCol() != SIZE) {
+            throw new IllegalArgumentException(Errors.format(ErrorKeys.ILLEGAL_MATRIX_SIZE));
+        }
+        if (matrix2.getNumRow() != SIZE || matrix2.getNumCol() != SIZE) {
+            throw new IllegalArgumentException(Errors.format(ErrorKeys.ILLEGAL_MATRIX_SIZE));
+        }
+        m00 = matrix1.getElement(0, 0) + matrix2.getElement(0, 0);
+    }
+    
+
+    @Override
+    public double determinate() {
+        return m00; // trivial 1x1 matrix
+    }
+
+    @Override
+    public void mul(double scalar) {
+        m00 *= scalar;
+    }
+
+    @Override
+    public void mul(double scalar, Matrix matrix) {
+        if (matrix.getNumRow() != SIZE || matrix.getNumCol() != SIZE) {
+            throw new IllegalArgumentException(Errors.format(ErrorKeys.ILLEGAL_MATRIX_SIZE));
+        }
+        m00 = scalar * matrix.getElement(0, 0);
+    }
+
+    @Override
+    public void mul(Matrix matrix) {
+        if (matrix.getNumRow() != SIZE || matrix.getNumCol() != SIZE) {
+            throw new IllegalArgumentException(Errors.format(ErrorKeys.ILLEGAL_MATRIX_SIZE));
+        }
+        m00 *= matrix.getElement(0, 0);
+    }
+
+    @Override
+    public void mul(Matrix matrix1, Matrix matrix2) {
+        if (matrix1.getNumRow() != SIZE || matrix1.getNumCol() != SIZE) {
+            throw new IllegalArgumentException(Errors.format(ErrorKeys.ILLEGAL_MATRIX_SIZE));
+        }
+        if (matrix2.getNumRow() != SIZE || matrix2.getNumCol() != SIZE) {
+            throw new IllegalArgumentException(Errors.format(ErrorKeys.ILLEGAL_MATRIX_SIZE));
+        }
+        m00 = matrix1.getElement(0, 0) * matrix2.getElement(0, 0);
+    }
+
+//    @Override
```

Comment:
Removed references to normalize and normalize( matrix ) we can implement later if needed.